To: vim_dev@googlegroups.com
Subject: Patch 7.3.417
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.417 (after 7.3.395)
Problem:    Test 43 fails with a tiny build.
Solution:   Only run test 43 with at least a small build.
Files:	    src/testdir/test43.in


*** ../vim-7.3.415/src/testdir/test43.in	2012-01-10 13:44:23.000000000 +0100
--- src/testdir/test43.in	2012-01-26 18:00:06.000000000 +0100
***************
*** 1,6 ****
--- 1,7 ----
  Tests for regexp with various magic settings.
  
  STARTTEST
+ :so small.vim
  :set nocompatible viminfo+=nviminfo
  /^1
  /a*b\{2}c\+/e
*** ../vim-7.3.415/src/version.c	2012-01-26 14:32:26.000000000 +0100
--- src/version.c	2012-01-26 18:02:30.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     417,
  /**/

-- 
I learned the customs and mannerisms of engineers by observing them, much the
way Jane Goodall learned about the great apes, but without the hassle of
grooming.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
